client/error.go

```diff
@@ -49,3 +49,12 @@ func (e VersionError) Error() string {
 	}
 	return fmt.Sprintf("%s running %s, %s", e.URI, e.V, e.Err)
 }
+
```

Comment:
this should be in `adaptor/mongodb/client.go` (below [here](https://github.com/compose/transporter/blob/master/adaptor/mongodb/client.go#L37-L44)) since it's specific to mongodb